backends/jgroups/src/test/resources/testing-flush-loopback.xml

```diff
@@ -7,14 +7,14 @@
   -->
 <config xmlns="urn:org:jgroups"
 		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/JGroups-3.6.xsd">
+		xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/jgroups-4.0.xsd">
 
 	<SHARED_LOOPBACK
-		bind_addr="127.0.0.1"
```

Comment:
I couldn't find the default for this, but I'll assume it's more or less `127.0.0.1`.